posts/key-learnings-from-waku-hackers-in-2023

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+name: 'Key Learnings from Waku Hackers in 2023'
+title: 'Key Learnings from Waku Hackers in 2023'
+date: 2023-12-20 00:00:00
+authors: amelia
+published: 
+slug: learnings-from-hackers
+image: /img/black-waku-logo-with-name.png
+---
+
+A review of the key learnings from Waku hackathon participants this year and what we are doing to improve developers' experience. 
+
+<!--truncate-->
+
+# Key Learnings from Waku Hackers in 2023
+
+2023 has been an incredible year for Waku. We attended eight events across four continents, participated in three ETHGlobal hackathons, and hosted the first Waku Hacker House. It has been amazing to see the growth of the developer community on Waku, and we want to make sure that next year it will be even easier to onboard new hackers and developers.
+
+We recently held an X Space where we invited the winning projects from the ETHRome hackathon to tell us what they like about Waku, what worked well while using Waku, and where they saw areas for improvement. Below, you will find some of the key insights from the hackers and learn how their feedback is informing improvements around Waku documentation and the greater developer experience. 
+
+[Václav](https://twitter.com/vpavlin), [Aaron](https://twitter.com/vrycmfy), and [Guru](https://twitter.com/hackyguru) represented the Waku team on the Space. We were also joined by participants from the winning ETHRome teams [Feka](https://twitter.com/Feka_7heaven), [mrnmrc.eth](https://twitter.com/mrnmrceth), [Martin](https://twitter.com/martinlecl), [Régis](https://twitter.com/Rgis295478), [Florian](https://twitter.com/floriantruchot), and [Mattia](https://twitter.com/Mattia).  
+
+[embed: https://twitter.com/Waku_org/status/1722659158816612463]
+
+
+## “Blockchain seems like the default way to do decentralised things, but in the end, it's not the only one” Mattia
```

Comment:
should there be an em dash between quote and author?

Reply:
You're correct, I've added this